docs/getting-started.mdx

```diff
@@ -57,11 +58,26 @@ Patrol UI tests. It is necessary to run UI tests (`flutter test` won't work! [He
    dart pub global activate patrol_cli
    ```
 
+  <Info>
+     Make sure to add `patrol_cli` to your `PATH` environment variable.
+  </Info>
+
```

Comment:
That's misleading – the binary is named `patrol` - and it's not explained how to do it. But it's [already explained in the README](https://pub.dev/packages/patrol_cli) - you can link to that place.